unittests/test-contracts/test_api/test_datastream.cpp

```diff
@@ -74,7 +74,24 @@ void test_datastream::test_basic()
         int a[2];
         bool operator==( const StaticArray &o ) const { return a[0] == o.a[0] && a[1] == o.a[1]; }
     };
-    testtype<StaticArray>::run( {{10,20}}, "StaticArray" );
+    /**
+     * Removed test:
+     * testtype<StaticArray>::run( {{10,20}}, "StaticArray" );
```

Comment:
Can we instead construct the StaticArray differently and keep the test or is brace elision the entire point of this test?

Based on that Github issue, the author has no intention on fixing it so we should either adjust the test or remove it outright.

Reply:
I spent a while trying to fix that. was experimenting with different sets of constructors but got no luck.
so to not spend many time on this I just commented this out.

Reply:
Yeah looking into this further, this has nothing to do with how it is initialized. It looks like its a bug in PFR when it counts the number of fields as part of the reflection. It's saying this is a size 2 when it should be a size 1.

I will spend some time investigating this, as it appears to be a possible regression in CDT.

Reply:
Yes, it is PFR bug that is based on braces elision. how would you determine number of elements in class with braces elision feature being active? Library author is still looking for suggestions.
I tried to experiment with adding some constructors like this to restrict braces elision but no luck:
```
struct my{
   int val[2];
   my(const int (&v)[2]) { }
};
int main ()
{
   boost::pfr::for_each_field<my>({{1,2}}, [](const auto& v){ cout << "kuku" << endl;});
}
```
in case above it doesn't find any constructor at all.
 I'll ask for suggestions in blockchain channel.